ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -116,6 +116,115 @@ internal object TestData {
         ),
         assetBaseURL = Constants.assetBaseURL,
     )
+    val template3 = PaywallData(
```

Comment:
I wonder if we should move each template test data to its own file, to avoid this file getting huge with all templates